src/opnsense/mvc/app/views/layout_partials/form_input_tr.volt

```diff
@@ -62,7 +62,7 @@
     </td>
     <td>
         {% if type == "text" %}
-            <input  type="text"
+            <input  type="text" aria-label="{{label}}"
```

Comment:
I’m not sure if translations are escaped already which would double-encode them here.

Reply:
@fichtner I'll doublecheck, at a first glance I expected it was unescaped.

Reply:
yep, escape seems to be needed.  replaced a label with the following

`<label><![CDATA["<strong>Color</strong>XXX]]></label>`

which renders without escape:
<img width="597" alt="image" src="https://github.com/opnsense/core/assets/9783985/b679a386-47cf-4ec9-9320-c528a678ec0a">

Reply:
My point is this https://github.com/opnsense/core/blob/9c16e16d8f772f2b1ee518640eb5a435568e2f92/src/opnsense/mvc/app/library/OPNsense/Base/ViewTranslator.php#L41

Reply:
Something is still fishy here. If it’s unescaped here for “label” the translation doesn’t execute properly (or at all) since it would have escaped already.

Reply:
Ok, maybe just issued via gettext()… but why don’t we escape the label itself in line 60 then?

Reply:
we don't need to escape it because it's inside a clause (and not an attribute or JS part), although we don't feed style blocks in labels, technically you are able to.